src/test/java/api/loans/AgedToLostScheduledNoticesProcessingTests.java

```diff
@@ -369,6 +370,11 @@ void patronNoticesForForAgedToLostFineAdjustmentsAreCreatedAndProcessed() {
     final ZonedDateTime cancelLostItemFeeActionDate = getActionDate(cancelLostItemFeeAction);
     final ZonedDateTime cancelProcessingFeeActionDate = getActionDate(cancelProcessingFeeAction);
 
+    scaleFields(refundLostItemFeeAction, "amountAction", "balance");
+    scaleFields(refundProcessingFeeAction, "amountAction", "balance");
+    scaleFields(cancelLostItemFeeAction, "amountAction", "balance");
+    scaleFields(cancelProcessingFeeAction, "amountAction", "balance");
```

Comment:
Why do we scale amounts in fee/fine actions? They should remain intact.

Reply:
removed